client/landing/stepper/declarative-flow/internals/steps-repository/plans/style.scss

```diff
@@ -188,5 +188,15 @@
 	.signup-header h1 {
 		display: none;
 	}
+
+	button.button.navigation-link.is-borderless {
+		background: none;
+		border: none;
+		color: #101517;
```

Comment:
Looks like this color is `--studio-gray-100`, can we reuse the variable here?

<img width="252" alt="image" src="https://user-images.githubusercontent.com/20927667/227630624-fd773b28-787c-43d0-8f63-508106e88c72.png">